### PR TITLE
Update: (TreePicker) Empty state on the left pane.

### PR DESCRIPTION
**Pull request by camjc**

### CODE REVIEW

**Comment by omgaz** [comment]
LGTM

File: test/components/adslotUi/TreePickerPureComponentTest.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import createComponent from 'helpers/shallowRenderHelper';
 import React from 'react';
 import TreePickerPureComponent from 'components/adslotUi/TreePickerPureComponent.js';
```
Comment (antklim) [inline test/components/adslotUi/TreePickerPureComponentTest.js]: Could you please demolish `.js` :cake:

File: test/components/adslotUi/TreePickerPureComponentTest.js [hunk 2]
```diff
@@ -28,7 +28,18 @@ describe('TreePickerPureComponent', () => {
 
     const nodesGridElement = leftPaneElement.props.children[2];
     expect(nodesGridElement.type).to.equal((<Grid />).type);
-    expect(nodesGridElement.props.children).to.have.length(0);
+
+    expect(nodesGridElement.props.children).to.have.length(2);
+    const nodesElements = nodesGridElement.props.children[0];
```
Comment (antklim) [inline test/components/adslotUi/TreePickerPureComponentTest.js]: I would add children' indexes into constants and use them across different test cases when necessary.

**Comment by antklim** [comment]
LGTM :radio:

**Comment by antklim** [comment]
Merging :rocket:

**Pull request merged**